llama.cpp

```diff
@@ -2162,7 +2163,11 @@ struct llama_vocab {
     std::unordered_map<token, id> token_to_id;
     std::vector<token_data>       id_to_token;
 
-    std::vector<id> special_tokens_cache;
+    bool has_cache = false;
+
+    std::vector<id> cache_special_tokens;
```

Comment:
I like this name change from `special_tokens_cache` to `cache_special_tokens`. 👍